person.rb

```diff
@@ -0,0 +1,32 @@
+require_relative './corrector'
+
+class Person
```

Comment:
- [ ] As stated in the requirements of this project, the person class should inherit from the `Nameable(Corrector)` class and it should also have a `correct_name` method. Please let's fix this and get closer to approval. You may consider doing something of this sort.


```ruby
class Person < Corrector
  .
  .

    def correct_name
       name
    end
  .
  .
end
```